tests/console/mysql_odbc.pm

```diff
@@ -36,9 +37,16 @@ sub setup {
     assert_script_run "echo Password=x >> /etc/unixODBC/odbc.ini";
 
     # write odbcinst.ini
-    assert_script_run "echo [myodbc_mysql] > /etc/unixODBC/odbcinst.ini";
+    assert_script_run "echo [mariadbodbc_mysql] > /etc/unixODBC/odbcinst.ini";
     assert_script_run "echo Description=ODBC for MySQL >> /etc/unixODBC/odbcinst.ini";
-    assert_script_run 'echo Driver=$(rpm --eval "%_libdir")/libmyodbc5.so >> /etc/unixODBC/odbcinst.ini';
+
+    if (!(is_sle && !sle_version_at_least('15')) && !(is_leap && !leap_version_at_least('15.0'))) {
+        assert_script_run 'echo Driver=$(rpm --eval "%_libdir")/libmaodbc.so >> /etc/unixODBC/odbcinst.ini';
+    }
+    else {
+        assert_script_run 'echo Driver=$(rpm --eval "%_libdir")/libmyodbc5.so >> /etc/unixODBC/odbcinst.ini';
```

Comment:
because the above two lines only differ in the filename I suggest to set this one in a variable, e.g.

```perl
my $lib = !(is_sle && !sle_version_at_least('15')) && !(is_leap && !leap_version_at_least('15.0')) ? 'maodbc' : 'myodbc5';
assert_script_run 'echo Driver=$(rpm --eval "%_libdir")/lib' . $lib . '.so >> /etc/unixODBC/odbcinst.ini';
```

WDYT?

Reply:
I thought about doing something like this. Personally I think it's less readable. Also because it kind of runs external commands. With the normal if I can faster see "ok in sle case execute this". But if you insist I'll change it tomorrow :)